qpc_tools/install/all.py

```diff
@@ -31,6 +33,55 @@ def __init__(self, subparsers):
         # pylint: disable=no-member
         CliCommand.__init__(self, self.SUBCOMMAND, self.ACTION,
                             subparsers.add_parser(self.ACTION))
+        self.parser.add_argument('--offline', dest='offline',
```

Comment:
Should we group `--offline` & `--offline-files` together so that you can pass in one without the other.

Reply:
yep

Reply:
Groups won't really enforce anything.  Just improve help.

```usage: PROG [--bar BAR] foo

group1:
  group1 description

  foo    foo help

group2:
  group2 description

  --bar BAR  bar help```